src/pysqldbm_cli/run.py

```diff
@@ -115,6 +114,35 @@ def get_alter_statement(
         print_json({"statement": statement})
 
 
+@run.command("get-latest-object-ddl")
+@click.option("--project-id", type=int, required=True)
+@click.option("--object-name", type=str, required=True)
+@click.option("--case-sensitive", is_flag=True, default=False)
+@click.option("--raw/--no-raw", default=False)
```

Comment:
Added `--case-sensitive` as a flag, which I think fits the spirit of the arg (it's optional in the API string too), but inconsistent with the `--raw/--no-raw` flag. Happy to change back for consistency?

Reply:
What makes it inconsistent? Do we need a warning or a check for mutually exclusive args?

Reply:
I'm new to CLI tools, or at least ones that are used by other people, so if there's a standard pattern to this I don't know it.
But, my feeling for it being incosistent was it's a true/false input in the same way that raw/not raw is. But whereas that's got a got a true and false input this is just a flag.
For mutually exclusive args, it's just raw/not raw right? I can add a check in my next PR (need to add the DBT yaml calls too). Is it work just picking one way and sticking to it though? Quite happy to change this to `--case-sensentive/--case-insensivite` args.

Reply:
oh I see now. Tbh, I've probably got the raw/no-raw wrong. There's some good stuff in here: https://click.palletsprojects.com/en/8.1.x/options/#boolean-flags

I think the way you've done it is right.